flask_elasticsearch.py

```diff
@@ -33,7 +33,11 @@ def __getattr__(self, item):
         ctx = stack.top
         if ctx is not None:
             if not hasattr(ctx, 'elasticsearch'):
-                ctx.elasticsearch = Elasticsearch(hosts=[ctx.app.config.get('ELASTICSEARCH_HOST')],
+                if type(ctx.app.config.get('ELASTICSEARCH_HOST')) is str:
```

Comment:
I like this, perhaps change it to `isinstance(host, basestring)` so that all strings are supported